BrainPortal/app/views/demands/index.js.erb

```diff
@@ -0,0 +1,26 @@
+<%
```

Comment:
I'm not sure what this new file is about... ? We're not using any ajax updating of the index table?